data/Kabit Savaiye Bhai Gurdas Ji/235.json

```
@@ -89,7 +89,7 @@
         "pronunciation_information": null,
         "type": "Pankti",
         "gurmukhi": {
-          "Seva Singh": "kUkr ijau cauc kwiF cwkI cwitby kau jwie; jw ky mITI lwgI dyKY qwhI pwCY DwveI [235["
+          "Seva Singh": "kUkr ijau cauc kwiF, cwkI cwitby kau jwie, jw ky mITI lwgI dyKY, qwhI pwCy DwveI ]235]"
```

```suggestion
          "Seva Singh": "kUkr ijau cauc kwiF; cwkI cwitby kau jwie; jw ky mITI lwgI dyKY; qwhI pwCy DwveI ]235]"
```

Same as above for vishram.